_events/2021/2021-06-community-call.md

```diff
@@ -0,0 +1,37 @@
+---
+title: June 2021 Community Call
+expires: 2021-06-10
+event_date: "June 10, 2021"
+layout: event
+repeated: false
+---
+
+The next US-RSE community call is on June 10th at 12ET/11CT/10MT/9PT.
+
+The main topic of discussion is
+**"Open Source Development"**
+
+This month we are hosting a call
+on the topic of [Open Source Development](https://github.com/USRSE/monthly-community-calls/issues/4).
+By chance this is the day after the first [Global Maintainer Summit](https://globalmaintainersummit.github.com) and
+three days after a [Upstream 2021](https://explore.tidelift.com/upstream/)! So it seems like a good week
+to have some conversations and strategizing around this topic.
+This call will be a chance to explore with others in the US-RSE community how we are 
+individually or in groups, utilizing, contributing to or maintaining open source software? 
+Some discussion topics will include, how is everyone using open-source, what tools are up and coming, what are
```

Comment:
```suggestion
Some discussion topics will include, how is everyone using open source, what tools are up and coming, what are
```
Hyphens seem a bit wonky. I might have put one in the "open source" above given it's been used adjectivally but, as we don't have one there, definitely don't think here.